src/passes/storage/storageVariableAccessRewriter.ts

```diff
@@ -56,12 +57,14 @@ export class StorageVariableAccessRewriter extends ASTMapper {
       error('VariableDeclaration.vType should be defined for compiler versions > 0.4.x'),
     );
 
-    const parent = node.parent;
-    const replacementFunc = ast.getUtilFuncGen(node).storage.read.gen(node, decl.vType);
-    ast.replaceNode(node, replacementFunc, parent);
-    // Do not recurse
-    // The only argument to replacementFunc is node,
-    // recursing would cause an infinite loop
+    if (decl.mutability !== Mutability.Constant) {
```

Comment:
Also here, if `isCairoConstant` is the check for which variables get printed as cairo constants, `!isCairoConstant` should be the check for which get processed here